code/game/objects/structures/crates_lockers/crates.dm

```diff
@@ -539,20 +539,23 @@
 		"3" = (100 - ((STOCK_RARE_PROB * rarity) + (STOCK_UNCOMMON_PROB * rarity)))
 	)
 
-	var/list/valid_loot = typesof(/obj/structure/closet/crate) - typesof(/obj/structure/closet/crate/secure/gear_loadout)
-	valid_loot -= typesof(/obj/structure/closet/crate/loot)
-	var/icontype = pick(valid_loot)
-	var/obj/structure/closet/crate/C = new icontype(get_turf(src))
+	var/icontype = pick(typesof(/obj/structure/closet/crate))
```

Comment:
Re-add the excluded gear loadout crates. Not particularly because they had anything in them, because all of them use the default secure crate icon. Which we don't want. We want unique icons to be used.

Reply:
I was mixing these up with the non-secure ones that had their own unique one, whoops. Done.